### DISCUSSION TITLE
HTTP proxy HTTPS? WebSocket connection?

### DISCUSSION THREAD

**Discussion by LazyYangi** [socketio/socket.io#3989] [Q&A]
socket.io.js:4500 WebSocket connection to 'wss://sc.io.gypserver.com/socket.io/?EIO=4&transport=websocket&sid=agaNW-f_HfJgRxESAAEN' failed: 

![微信截图_20210625120350](https://user-images.githubusercontent.com/30524761/123367948-7392ef80-d5ad-11eb-86c2-dc70e1061658.png)

**Comment by darrachequesne** [comment]
The server does seem reachable:

```
$ curl -v "https://sc.io.gypserver.com/socket.io/?EIO=4&transport=polling"
0{"sid":"Mg_YFw66vy2wd6R6AAEk","upgrades":["websocket"],"pingInterval":25000,"pingTimeout":20000}
```

Are you using multiple Socket.IO servers?

Documentation:

- https://socket.io/docs/v4/troubleshooting-connection-issues/
- https://socket.io/docs/v4/using-multiple-nodes/
